src/main/resources/reference.conf

```diff
@@ -190,6 +190,9 @@ cassandra-journal {
   # Dispatcher for the plugin actor.
   plugin-dispatcher = "cassandra-journal.default-dispatcher"
 
+  # The query journal to use when recoverying
+  recovery-cassandra-query-journal = "cassandra-query-journal"
+
```

Comment:
This is a great catch. It must be one-to-one between the write and the query plugins. It would be more elegant if we could enforce that in the configuration. I'm thinking if we can do that the query journal is always a sub-section of the write journal config.

```
cassandra-journal {
  query-journal {
  }
}

other-cassandra-journal {
  query-journal {
  }
}
```

Then we would not need these cross references.

To implement that we need: https://github.com/akka/akka/issues/19822

Your pull request is a good first step (to make it work). What do you think about naming the property `query-plugin` to align it with the existing `write-plugin`: https://github.com/pbarron/akka-persistence-cassandra/blob/master/src/main/resources/reference.conf#L432 ?